docs/cloud-native-security/get-started-with-kspm.asciidoc

```diff
@@ -0,0 +1,186 @@
+
+[discrete]
+[[kspm-use-temp-credentials]]
+===== Option 2 - Use temporary security credentials:
+Temporary security credentials can be configured in AWS to last for some period of time. They consist of an access key ID, a secret access key, and a security token, which is typically found using `GetSessionToken`.
+
+NOTE: IAM users with multi-factor authentication (MFA) enabled need to submit an MFA code when calling `GetSessionToken`. For more details refer to https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[AWS Temporary Security Credentials].
+
+You can use the AWS CLI to generate temporary credentials. For example, you could use the following command if you have MFA enabled: `sts get-session-token --serial-number arn:aws:iam::1234:mfa/your-email@example.com --duration-seconds 129600 --token-code 123456`
+
+The output from this command should include the following fields, which you should supply to the KSPM integration:
+
+`Access key ID`: The first part of the access key.
+`Secret Access Key`: The second part of the access key.
+`Session Token`: A token required when using temporary security credentials.
+
+Because temporary security credentials are short term, once they expire you will need to generate new ones and manually update the integration's configuration to continue collecting cloud posture data. Update the credentials before they expire to avoid data loss.
+
+[discrete]
+[[kspm-use-a-shared-credentials-file]]
+===== Option 3 - Use a shared credentials file:
+If you use different AWS credentials for different tools or applications, you can use profiles to define multiple access keys in the same configuration file. For more details refer to https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html#file-format-creds[Create Shared Credentials File]
+
+Instead of providing the `Access key ID` and `Secret Access Key` to the integration, you will provide the information required to locate the access keys within the shared credentials file:
+
+`Credential Profile Name`: The profile name in the shared credentials file.
+`Shared Credential File`: The directory of the shared credentials file.
+
+If you don't provide values for all configuration fields, the integration will use these defaults:
+
+- If none of `Access key ID`, `Secret Access Key` and `ARN Role` are provided, then the integration will check for `Credential Profile Name`.
+- If there is no `Credential Profile Name`, the default profile will be used.
+- If `Shared Credential File` is empty, the default directory will be used.
+  - For Linux or Unix, the shared credentials file is located at `~/.aws/credentials`.
+
+IMPORTANT: If you choose this option, make sure the associated role has the permissions listed above.
+
+[discrete]
+[[kspm-use-iam-arn]]
+===== Option 4 - Use an IAM role Amazon Resource Name (ARN):
+An IAM role ARN is an IAM identity that you can create in your AWS account. You define the role's permissions.
```

Comment:
I think it helps to spell out the acronym in body text too. I don't think we typically do that only in section headings
```suggestion
An IAM role Amazon Resource Name (ARN) is an IAM identity that you can create in your AWS account. You define the role's permissions.
```

Also, as a whole this paragraph is pretty long -- consider breaking it up somehow.